### ISSUE TITLE
Journal etc. outside Trilium Demo tree

### ISSUE THREAD

**Issue by rubaboo** [zadam/trilium#1036]
I would like Today, New Task, and Weight Tracker entries appear someplace outside the Trilium Demo tree. Also, I would like all the functionality behind those (as well as Steel Blue theme) stored independently from the Demo. How can I achieve that?

In other words, I would like to have
1. A read-only copy of Demo
2. A copy of Demo to play with and trash
3. Journal, Task, and Weight Tracker inside a top-level note that I create (or if I could import that from somewhere, without any "filler" content, that would be great too).

Hope this is not too confusing..

**Comment by rubaboo** [comment]
After poking around some more, I think what I want is

1. A copy of Demo with all the functionality, but none of the "filler" content. "Functionality" means buttons, styles, scripts, forms, lookup values and whatever else that makes Trilium Demo work the way it does out of the box.
2. Nice to have: A way to also keep a full Demo but with all the functionality disabled.

**Comment by zadam** [comment]
Hello, if you like some of the stuff in the Trilium Demo and want to use it yourself, then you can just move it out (drag & drop, ctrl + x, ctrl + v) of the "Trilium Demo" subtree and put it wherever you want. Then you can delete the rest - the stuff you don't want ...

**Comment by rubaboo** [comment]
Fair enough. It's just not immediately obvious which parts play a special role and which don't. Could you list all the special nodes or provide a search string to identify all of them?

**Comment by eliandoran** [comment]
The Journal has been moved outside the Trilium Demo for new users.
